src/dmd/lexer.d

```diff
@@ -806,7 +806,7 @@ struct Lexer(R) if (isInputRange!R)
                      // \u or \U could be start of identifier
                     src.popFront();
                     err_fatal("unrecognized preprocessor token x%02x", c);
-                    assert(0);   // not handled yet
+                    //assert(0);   // not handled yet
                     break;
```

Comment:
delete line 810 and leave 809 uncommented

Reply:
Changed as requested.